### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by elmar8287**
- Add contact form in HTML
- Create styles for form as in template

### CODE REVIEW

**Review by Temmarie — APPROVED** [review]
## Status: Approved ✔️ 
### Hi @elmar8287 and @SimonGrchevski , 
#### Good job implementing the project's requirements 👍🏾 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
#### Congratulations! 🎉 

#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Review by mricanho — COMMENTED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @elmar8287!

Nothing to add here! Your project looks amazing and the code it's clean and works! 

shine team

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/lynters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x stylelint-csstree-validator
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (usmansbk) [inline .github/workflows/lynters.yml:48]: You don't have to type the linter configuration manually. You can copy the one from the Microverse linter documentation [here](https://github.com/microverseinc/linters-config/tree/master/html-css) and add all the files to your project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+                <li>
+                    <a href="#"><img src="images/icons/medium.svg" alt="mb" class="social-icon" /></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/icons/angel-list.svg" alt="xxx" class="social-icon" /></a>
+                </li>
+            </ul>
+        </section>
+    </div>
+    <section class="works grid-container">
```
Comment (hiromataba) [inline index.html:46]: # I think it would be nicer to put your work in the main tag

**Review by hiromataba — COMMENTED** [review]
# Your Project looks awesome and please read my comment down below 

# Thank you

File: .github/workflows/lynters.yml (hunk 1 above)
Comment (elmar8287) [reply .github/workflows/lynters.yml:48]: Dear @usmansbk,

Thanks for your comment, I will consider it for my future projects.

My regards,

Elmar

File: index.html (hunk 2 above)
Comment (elmar8287) [reply index.html:46]: Dear @hiromataba,

Yes, you are right, I did not use the main tag in this project. I will review my HTML file and fix it.

My regards,

Elmar

**Pull request merged**